content/_index.html

```diff
@@ -7,7 +7,7 @@
     <div class="wrap grid-one">
       <div>
         <h1 style="color: white;">Kubernetes Dynamic Admission at your fingertips</h1>
-				<h4 style="color: white;">Flexible, secure and portable thanks to WebAssembly. Get started with the latest version - <a href="#get-started">1.13</a></h4>
+				<h4 style="color: white;">Flexible, secure and portable thanks to WebAssembly. Get started with the latest version - <a href="#get-started">1.12</a></h4>
```

Comment:
Why do we downgrade here?

Reply:
My idea was to have something to update, to check the process worked. I can take it out.

Reply:
reset to 1.13